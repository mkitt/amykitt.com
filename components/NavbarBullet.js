import React from 'react'
import css from '../styles/css'

const style = css({
  fontSize: 12,
  fontWeight: 700,
  paddingRight: 10,
  paddingLeft: 10,
  textTransform: 'uppercase',
})

export default() => (
  <span className={style}>&bull;</span>
)
