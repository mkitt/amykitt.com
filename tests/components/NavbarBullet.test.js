import React from 'react'
import renderer from 'react-test-renderer'
import NavbarBullet from '../../components/NavbarBullet'

describe('NavbarBullet', () => {
  let component

  beforeEach(() => {
    component = <NavbarBullet />
  })

  it('shows the correct snapshot for the NavbarBullet component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
