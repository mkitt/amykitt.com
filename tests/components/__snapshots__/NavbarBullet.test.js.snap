// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarBullet shows the correct snapshot for the NavbarBullet component tree 1`] = `
<span
  className={
    Object {
      "data-css-3eixqd": "",
    }
  }
>
  â€¢
</span>
`;
